tmf/org.eclipse.tracecompass.tmf.core/src/org/eclipse/tracecompass/tmf/core/dataprovider/IDataProviderDescriptor.java

```diff
@@ -61,6 +61,11 @@ public enum ProviderType {
          * @since 6.1
          */
         DATA_TREE,
+        /**
+         * A provider for generic xy charts with a time-less x-axis
+         * @since 10.2
+         */
+        TREE_GENERIC_XY,
```

Comment:
should be moved to the end, otherwise it's API breaking and not just API addition.